lib/beaker-hostgenerator/data.rb

```diff
@@ -1098,6 +1098,11 @@ def generate_osinfo
         yield ["freebsd#{release}-64", "freebsd-#{release}-amd64"]
       end
 
+      # Amazon Linux >= 2023
+      (2023..2029).each do |release|
+        yield ["amazon#{release}-64", "amazon-#{release}-x86_64"]
+      end
```

Comment:
This will also generated the even numbers, which won't exist. 2025 and beyond also doesn't exist today, so I'd stick to just 2023 now.
```suggestion
      # Amazon Linux
      yield ["amazon2023-64", "amazon-2023-x86_64"]
```

Reply:
Ha, oops thanks ya I'll just go with that then.